pages/LoginPage.tsx

```diff
@@ -18,10 +19,7 @@ export default function LoginPage() {
           <Input type="text" placeholder="Benutzername" />
           <Input type="password" placeholder="Passwort" />
         </div>
-        <Button
-          label="Login"
-          onClick={() => alert("next page is coming soon")}
-        />
+        <Button label="Login" onClick={() => router.push("/ProfilePage")} />
```

Comment:
I would write filenames and routes in small letters

```suggestion
        <Button label="Login" onClick={() => router.push("/profilepage")} />
```